engineering/workflow/1-week-iteration-worklflow.md

```diff
@@ -0,0 +1,101 @@
+
+**Principle 1**; General statement that is true 90% of the time: Better to always move forward than not moving forward once we are moving forward in the direction of the Iteration/Team goal. Make a decision and communicate it. The exception is busy work and unplanned work - that will kill us - better to help someone else. Move the team forward. If you can’t, move a member of the team forward.
+
+## Process changes
+
+The introduction of the 1-week iterations address the following:
+1. How to prevent waiting
+1. What are the queue on the board and their purposes
+1. How do deal with getting block
+1. Process time-box and evaluation
+1. How will the ceremonies change
+
+### 1. How to prevent waiting
+
+1. Ask for help. (engineering slack)
+1. Refer to **Principle 1** if help cannot come immediately
+    1. Pull something new from the committed queue
+    1. Help someone else (who is already working on a tasks)
+    1. Avoid unplanned work
+    1. Raise this in the retrospective
+
+### 2. What are the queue on the board and their purposes
+
+1. The team will continue to use the [github project board](https://github.com/orgs/2i2c-org/projects/49/views/1) to manage their day to day tasks.
+
+1. For the next 2-months the engineering process will use 1-week sprints.
+
+1. To support partnerships, the product allocation bucket will be reduced based on needs to support new grant requests.
+
+1. Work that is not related to grant/funding raising will follow the normal product process
+
+1. Work that is in the committed queue is expected to be completed by the end of the iteration **unless** new unplanned work is introduced from support requests in the grant writing process.
+
+1. In the unlikely event that an engineer has ran out of work, find a way to support another engineer to complete their tasks, pull work from the committed queue, or make a decision on work to do next based on **Principle 1**. Note: Our goal is to reduce unplanned work. However, if we are forced to introduce unplanned work, this should be made clear to the team and should considered a topic of discussion in the subsequent retrospective.
+
+### 3. How do deal with getting block
+
+Getting blocked happens for various reasons. The most common is related to _unforeseen or poorly managed dependencies_. The guidelines for dealing with impediments (i.e. blockers) is:
+1. Use the Engineering slack channel to ask for help (async)
+1. Avoid unplanned work (see Principle 1) and [Accidental complexity](https://www.nutshell.com/blog/accidental-complexity-software-design).
+1. We'll adopt Sync check-ins (15 minutes). This will help to create alignment in the team on what's the most important tasks to work on. The check-ins are just to surface progress, we we plan to work on next, highlight things that can derail delivery. Deep-dives are addressed through conversations after the sync (e.g. “can we talk afterwards?”)
```

Comment:
```suggestion
1. We'll adopt Sync check-ins (15 minutes). This will hel create alignment in the team on what's the most important tasks to work on. The check-ins are just to surface progress, what we plan to work on next, highlight things that can derail delivery. Deep-dives are addressed through conversations after the sync (e.g. “can we talk afterwards?”)
```

Reply:
I think there's a typo in your suggestion 😅